fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -48,17 +49,16 @@ def __init__(self, X, sensitive_features, y, estimator, constraints, eps, B, opt
         self.eps = eps
         self.B = B
         self.opt_lambda = opt_lambda
-        self.hs = pd.Series(dtype="float64")
-        self.classifiers = pd.Series(dtype="float64")
-        self.errors = pd.Series(dtype="float64")
-        self.gammas = pd.DataFrame()
-        self.lambdas = pd.DataFrame()
-        self.n = self.X.shape[0]
-        self.n_oracle_calls = 0
-        self.n_oracle_calls_dummy_returned = 0
-        self.oracle_execution_times = []
-        self.last_linprog_n_hs = 0
-        self.last_linprog_result = None
+        self.hs_ = pd.Series(dtype="float64")
```

Comment:
I don't think that we need to be using this convention in classes that are not estimators. I'd rather revert these.

Reply:
Sounds good!